addons/website_event_track/controllers/event_track.py

```diff
@@ -448,6 +448,8 @@ def send_email_reminder(self, track_id, email_to):
             error_message = _('The talk is already finished.')
         elif not track.is_track_upcoming:
             error_message = _('The talk has already begun.')
+        elif not track.date:
```

Comment:
Would move before "is_track_upcoming", as condition of having a date should come before a condition based on date :)